core/frontend.ml

```diff
@@ -66,7 +66,7 @@ struct
        ->- after_typing_ext session_exceptions ((DesugarSessionExceptions.insert_toplevel_handlers tyenv)#program ->- snd3)
        ->- after_typing_ext session_exceptions ((DesugarSessionExceptions.desugar_session_exceptions tyenv)#program ->- snd3)
        ->- after_typing ((DesugarProcesses.desugar_processes tyenv)#program ->- snd3)
-       ->- after_typing ((DesugarDbs.desugar_dbs tyenv)#program ->- snd3)
+       (*->- after_typing ((DesugarDbs.desugar_dbs tyenv)#program ->- snd3)*)
```

Comment:
Can this just be deleted, along with the file itself?

Reply:
Yes, that's the plan.